### PR TITLE
12.0 remove beesdoo product dependency

### PR DESCRIPTION
**Pull request by victor-champonnois**
## Description
related to https://github.com/beescoop/Obeesdoo/blob/12.0/beesdoo_product/views/beesdoo_product.xml

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

**Comment by carmenbianca** [comment]
@victor-champonnois I think you did some git stuff wrong. There's a lot of foodhub_munich_custom stuff in this PR.

**Comment by victor-champonnois** [comment]
@carmenbianca ah yes thanks, it is fixed

File: setup/foodhub_munich_custom/odoo/addons/foodhub_munich_custom [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../foodhub_munich_custom
```
Comment (carmenbianca) [inline setup/foodhub_munich_custom/odoo/addons/foodhub_munich_custom]: Can you delete this?

File: setup/foodhub_munich_custom/setup.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import setuptools
+
+setuptools.setup(
+    setup_requires=['setuptools-odoo'],
+    odoo_addon=True,
+)
```
Comment (carmenbianca) [inline setup/foodhub_munich_custom/setup.py]: Idem.

File: spp_custom/__manifest__.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "SPP Customizations",
     "version": "12.0.1.1.0",
-    "depends": ["beesdoo_base", "beesdoo_product"],
+    "depends": ["beesdoo_base", "product"],
```
Comment (carmenbianca) [inline spp_custom/__manifest__.py]: Can you do the dependency list like so?

```python
"depends": [
    "beesdoo_base",
    "product",
],
```

It reduces conflicts like the one we had.

Also, are you sure `beesdoo_base` is a correct dependency here? It's an emptied module. Can we change it instead to some of the dependencies of `beesdoo_base` itself?

**Review by carmenbianca — CHANGES_REQUESTED** [review]

File: setup/foodhub_munich_custom/odoo/addons/foodhub_munich_custom (hunk 1 above)
Comment (victor-champonnois) [reply setup/foodhub_munich_custom/odoo/addons/foodhub_munich_custom]: done

File: setup/foodhub_munich_custom/setup.py (hunk 2 above)
Comment (victor-champonnois) [reply setup/foodhub_munich_custom/setup.py]: done

File: spp_custom/__manifest__.py (hunk 3 above)
Comment (victor-champonnois) [reply spp_custom/__manifest__.py]: removed beesdoo_base
reformatted the dependencies but precommit does not let me... so I forced it. Weird...

Comment (carmenbianca) [reply spp_custom/__manifest__.py]: You need to include a trailing comma after the second item. See `"product",`

**Review by carmenbianca — APPROVED** [review]
LGTM thank you so much victor

**Pull request merged**